app/controllers/works_controller.rb

```diff
@@ -41,6 +43,12 @@ def edit
   end
 
   def update
+    unless @work.user_id == @login_user.id
+      flash[:status] = [:failure]
+      flash[:result_text] = "Only the owner of the work can edit it."
```

Comment:
These lines are repeated frequently in this file. Would it be possible to DRY them up with a controller filter?